collectors/30/kafka_offset_collector.py

```diff
@@ -16,103 +21,166 @@ def format_tsd_key(metric_key, metric_value, timestamp, tags={}):
-        zk = KazooClient(hosts=zk_quorum)
-        kafka = KafkaClient(kafka_brokers)
+    consumer_groups = ['secor_group', 'storm']
 
+    for kafka in kafkas.split('|'):
+        env, kafka, zk = parse_kafka_info(kafka)
         zk.start()
+        topics = get_kafka_topics(zk)
         for topic in topics:
-            total_broker_offset = report_broker_info(kafka, zk, topic)
-            total_consumer_offset = 0
-            for partition in get_partitions(zk, consumer_group, topic):
-                offset = get_consumer_group_offset(zk, consumer_group, topic, partition)
-                total_consumer_offset += offset
-                print format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offset)
-            print format_kafka_consumer_lag_tsd_key(consumer_group, topic, total_broker_offset - total_consumer_offset)
+            total_broker_offset = report_broker_info(env, kafka, zk, topic)
+            for consumer_group in consumer_groups:
+                total_consumer_offset = 0
+                for partition in get_partitions(zk, topic):
+                    offset = get_consumer_group_offset(zk, consumer_group, topic, partition)
+                    total_consumer_offset += offset
+                    print format_kafka_consumer_offset_tsd_key(env, consumer_group, topic, partition, offset)
+                print format_kafka_consumer_lag_tsd_key(env, consumer_group, topic, total_broker_offset - total_consumer_offset)
         zk.stop()
-
         kafka.close()
         zk.close()
 
 
+class StormPaths:
+    storm_chroot = '/storm-transactional/raw_events_stream/user'
+
+    @staticmethod
+    def partition(partition):
+        return '{root}/partition_{partition}'.format(root=StormPaths.storm_chroot, partition=partition)
+
+    @staticmethod
+    def batch(partition, batch):
+        return '{partition}/{batch}'.format(partition=StormPaths.partition(partition), batch=batch)
+
+
 class KafkaPaths:
     kafka_chroot = '/kafka'
 
     @staticmethod
     def topic_offsets(group, topic):
-        return '%s/consumers/%s/offsets/%s' % (KafkaPaths.kafka_chroot, group, topic)
+        return '{root}/consumers/{group}/offsets/{topic}'.format(root=KafkaPaths.kafka_chroot, group=group, topic=topic)
 
     @staticmethod
     def consumer_topic_partition(group, topic, partition):
-        return KafkaPaths.topic_offsets(group, topic) + '/' + partition
+        return '{topic}/{partition}'.format(topic=KafkaPaths.topic_offsets(group, topic), partition=partition)
+
+    @staticmethod
+    def topics():
+        return '{root}/brokers/topics'.format(root=KafkaPaths.kafka_chroot)
 
     @staticmethod
-    def broker_partitions(topic):
-        return '%s/brokers/topics/%s/partitions' % (KafkaPaths.kafka_chroot, topic)
+    def partitions(topic):
+        return '{topics_path}/{topic}/partitions'.format(topics_path=KafkaPaths.topics(), topic=topic)
 
 
 if __name__ == '__main__':
+    global storm_zk
```

Comment:
Different places in this file we switch back and forth between production zk and staging, is there any chance we can make that more explicit?